examples/gno.land/p/demo/grc/grc20/token.gno

```diff
@@ -127,6 +127,10 @@ func (led *PrivateLedger) Transfer(from, to std.Address, amount int64) error {
 	led.balances.Set(string(to), newToBalance)
 	led.balances.Set(string(from), newFromBalance)
 
+	if newFromBalance == 0 {
+		led.balances.Remove(string(from))
+	}
+
```

Comment:
no need to do `led.balances.Set(string(from), newFromBalance)` if newFromBalance is zero?

Reply:
Done https://github.com/gnolang/gno/pull/4420/commits/6c7264efc88aebdb638f660bebc9858d89c45cc8